test/test_other.py

```diff
@@ -13729,6 +13729,15 @@ def test_no_minify(self):
     content = read_file('a.out.js')
     self.assertContained(comment, content)
 
+  def test_no_minify_and_later_closure(self):
```

Comment:
IIRC that only what to get closure to work after the linker is don't is with `--minify=0` .. so the `no_minify` part of this test name seems redunant.

Can we just call it something like `test_closure_after_link`?    (And if you like add a comment saying "--minify=0  is the currently required to run closure on the emscripten output").

Reply:
Happy either way, but I did it this way for consistency with the last test right before it, whose name is `test_no_minify`. This test is really an extension of it - I considered just appending to it, actually.

Reply:
I see.  Given the name of the previous test I guess this is fine.